### PR TITLE
dumpxml: Case update according to libvirt change

### PR DESCRIPTION
**Pull request by will-Do**
libvirt commit 3b6be3c0 rework cpuUpdate, so the expect result also need
update.
This patch also rework the test case:
1). Skip case early when check libvirt version for ACL testing;
2). Simplify running command and checking test result part;
3). Remove autotest dependency;
4). Add more debug messages.

Signed-off-by: Yanbing Du <ydu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py [hunk 1]
```diff
@@ -143,8 +156,21 @@ def check_cpu(xml, cpu_match):
...
     vm_state = params.get("dumpxml_vm_state", "running")
     security_pwd = params.get("dumpxml_security_pwd", "123456")
-    status_error = params.get("status_error", "no")
+    status_error = "yes" == params.get("status_error", "no")
     cpu_match = params.get("cpu_match", "minimum")
+
+    # acl polkit params
+    setup_libvirt_polkit = "yes" == params.get('setup_libvirt_polkit')
+    if not libvirt_version.version_compare(1, 1, 1):
+        if setup_libvirt_polkit:
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py]: if `setup_libvirt_polkit` is checked first and then for `libvirt_version.version_compare` then we can avoid calling `libvirt_version` for other non acl tests.

Comment (waynesun09) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py]: 1. version_compare only return True or False
2. another compare happen in check_cpu function at line 118

save the result not apply

File: libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py [hunk 2]
```diff
@@ -143,8 +156,21 @@ def check_cpu(xml, cpu_match):
...
+            raise exceptions.TestSkipError("API acl test not supported in"
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py]: let us start using test.cancel()

File: libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py [hunk 3]
```diff
@@ -153,22 +179,10 @@ def check_cpu(xml, cpu_match):
         try:
             vm_xml.VMXML.add_security_info(new_xml, security_pwd)
         except Exception, info:
-            raise error.TestNAError(info)
+            raise exceptions.TestSkipError(info)
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py]: same here and elsewhere

Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py]: @balamuruhans 
So you are using avocado latest version(not the 36lts)?
As we are using 36lts version for daily testing, it support skip status and don't backport the cancel status, so we have to use ```skip``` for now. I will raise a new patch to backport it.

Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/domain/virsh_dumpxml.py]: @will-Do yes I am using upstream avocado.
sure. I agree

**Comment by balamuruhans** [comment]
Looks good, Thanks @will-Do

**Pull request merged**